Project.toml

```
@@ -50,7 +50,7 @@ ForwardDiff = "0.10.3"
 Libtask = "0.4, 0.5.3"
 MCMCChains = "4"
 NamedArrays = "0.9"
-Reexport = "0.2, 1"
+Reexport = "< 1.2.1"
```

This reintroduces compatibility with Reexport 0.1. We should better state explicitly
```suggestion
Reexport = "0.2, ~1.0, ~1.1, =1.2.0"
```
(or something similar)

---

I realize but don't you think that doesn't matter at all? As in nobody uses Reexport@0.1.

---

But people that still use it might be offered this version which (it seems) is actually not compatible.

---

True, but how long do we keep that compat entry? Like, in 10 years, do we still have it around because someone might try to do that? :sweat_smile: 

I'll commit your change though, np :+1: